build/scripts/tests.sh

```diff
@@ -39,7 +39,7 @@ mkdir -p "${log_dir}"
 for test_path in "${unittest_dir}"/*/"${target_framework}"
 do
     publish_test_path="${test_path}"/"${runtime_id}"/publish
-    if [ -d "${publish_test_path}" ]
+    if [[ -d "${publish_test_path}" ]]
```

Comment:
what magic is this?

Reply:
Single brace is a process invocation (literally, there's a file called `/usr/bin/[`), it's the same executable as `test` iirc.

Double braces is a shell builtin that does mostly the same thing as the executable (iirc it uses its special status to be a bit more sane, too). The reason why I changed this is to just save a fork/exec.